sim/core/item_swaps.go

```diff
@@ -268,9 +282,19 @@ func (swap *ItemSwap) GetUnequippedItemBySlot(slot proto.ItemSlot) *Item {
 }
 
 func (swap *ItemSwap) ItemExistsInMainEquip(itemID int32, possibleSlots []proto.ItemSlot) bool {
+	if !swap.IsEnabled() {
+		return swap.character.Equipment.containsItemInSlots(itemID, possibleSlots)
+	}
 	return swap.originalEquip.containsItemInSlots(itemID, possibleSlots)
 }
 
+func (swap *ItemSwap) EnchantExistsInMainEquip(effectID int32, possibleSlots []proto.ItemSlot) bool {
+	if !swap.IsEnabled() {
+		return swap.character.Equipment.containsEnchantInSlots(effectID, possibleSlots)
+	}
+	return swap.originalEquip.containsEnchantInSlots(effectID, possibleSlots)
+}
```

Comment:
I don't think you need these additions anymore after the refactor?

Reply:
Yeah it looks like `ItemExistsInMainEquip()`, `ItemExistsInSwapEquip()`, and `EnchantExistsInMainEquip()` are not called anywhere in the codebase after your changes, so all 3 can be deleted now.